Reference/google/protobuf/any.pb.swift

```diff
@@ -108,18 +108,14 @@ import Foundation
 ///         "@type": "type.googleapis.com/google.protobuf.Duration",
 ///         "value": "1.212s"
 ///       }
-struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message {
+struct Google_Protobuf_Any: ProtobufGeneratedMessage, ProtobufProto3Message, ProtoNameProviding {
   public var swiftClassName: String {return "Google_Protobuf_Any"}
   public var protoMessageName: String {return "Any"}
   public var protoPackageName: String {return "google.protobuf"}
-  public var jsonFieldNames: [String: Int] {return [
-    "typeUrl": 1,
-    "value": 2,
-  ]}
-  public var protoFieldNames: [String: Int] {return [
-    "type_url": 1,
-    "value": 2,
-  ]}
+  public static let _protobuf_fieldNames: FieldNameMap = [
+    1: .unique(proto: "type_url", json: "typeUrl", swift: "typeURL"),
+    2: .same(proto: "value", swift: "value"),
```

Comment:
anecdotal from ObjC protobuf's history - -description used to print out the objc names as it logged the message.  That output would make it into emails/bug reports/etc, and would some times confuse the folks that worked server side or on other platforms as the fields didn't fully line up to what they expected.  When we switch to using the TextFormat names, all the time spent saying, "no, that's what is in the message, they are just the objc names" went away as the protos were what all developers expected since the format is programming language agnostic.  So I sorta see #149 as helping apply that same logic.